api/views.py

```diff
@@ -59,6 +69,43 @@ def index():
     """ route to index of the API. """
     return jsonify({'Home': 'Index of the API', "Docs":"/apidocs"})
 
+@jwt.unauthorized_loader
+def unauthorized_response(callback):
+    return jsonify({
+        'ok': False,
+        'message': 'Missing Authorization Header'
+    }), 401
+
+
+def admin_token_required(f):
+    @wraps(f)
+    def decorated(*args, **kwargs):
+
+        # data, status = get_logged_in_user(request)
```

Comment:
Keep your code base clean.

Reply:
I integrated PEP8 to my IDE, so I will be sure to follow the standards in the coming PRs. Thank you.